content/workshops/rhel_8/exercise1.0.adoc

```diff
@@ -0,0 +1,236 @@
+
+The logs module provides date-sorted view of recent system log entries.  You can change the date or apply basic filters using the pull-down menus at the top of the page. 
+
+=== Section 4: Storage Module
+
+image::cockpit_02_storage.png[caption="Figure 4: ", title='Web Console Storage Module', 500]
+
+The storage module provides graphs of current storage performance.  It also lists local and network-mounted filesystems, plus recent storage-related log entries.
+
+=== Section 5: Networking Module
+
+image::cockpit_03_networking.png[caption="Figure 5: ", title='Web Console Networking Module', 500]
+
+The network module contains performance graphs similar to the storage module.  Additionally, it shows the configuration of your network interfaces, identifies unused interfaces, and provides access to, and control of, the system firewall.
+
+=== Section 5: Podman Containers Module
+
+image::cockpit_04_podman.png[caption="Figure 6: ", title='Web Console Podman Containers Module', 500]
+
+This module provides control of local podman containers and their corresponding images.
+
+=== Section 6: Virtual Machines Module
+
+image::cockpit_05_vm.png[caption="Figure 7: ", title='Web Console Virtual Machines Module', 500]
+
+The main screen that you see, when you first login to the web console is the system module.  It contains an overview of the system, including the below sections:
+
+=== Section 7: Accounts Module
+
+image::cockpit_06_accounts.png[caption="Figure 8: ", title='Web Console Accounts Module', 500]
+
+The accounts module allows you to create and modify local user accounts.
+
+=== Section 8: Services Module
+
+image::cockpit_07_services.png[caption="Figure 9: ", title='Web Console Services Module', 500]
+
+The services module allows basic administration of systemd objects, including system services.
+
+=== Section 9: Session Recording Module
+
+image::cockpit_08_sr.png[caption="Figure 10: ", title='Web Console Session Recording Module', 500]
+
+This module provides management of recorded terminal sessions. 
+
+=== Section 10: Applications Module
+
+image::cockpit_09_applications.png[caption="Figure 11: ", title='Web Console Applications Module', 500]
+
+The main screen that you see, when you first login to the web console is the system module.  It contains an overview of the system, including the below sections:
+
+=== Section 11: Diagnostic Reports Module
+
+image::cockpit_10_dr.png[caption="Figure 12: ", title='Web Console Diagnostic Reports Module', 500]
+
+The main screen that you see, when you first login to the web console is the system module.  It contains an overview of the system, including the below sections:
+
+=== Section 12: Kernel Dump Module
+
+image::cockpit_11_kd.png[caption="Figure 13: ", title='Web Console Kernel Dump Module', 500]
+
+The main screen that you see, when you first login to the web console is the system module.  It contains an overview of the system, including the below sections:
```

Comment:
Line 177 The system module is the main screen that displays upon logging in to the web console. It presents an overview of the system, including the following system characteristics: